### PR TITLE
feat: Don/avatars

### PR DESCRIPTION
**Pull request by BryanPan342**
@DonLe27

### CODE REVIEW

File: server/src/index.js [hunk 1]
```diff
@@ -2,73 +2,72 @@ const http = require('http');
 const express = require('express');
 const socketIo = require('socket.io');
```
Comment (BryanPan342) [inline server/src/index.js]: plox turn off ur auto style LMAO

**Review by BryanPan342 — COMMENTED** [review]
plox remove autostyle

File: server/src/index.js [hunk 2]
```diff
@@ -2,73 +2,72 @@ const http = require('http');
...
+
+            if (occupants == {}) {
+                logger.info('Everyone left');
+                delete rooms[curRoom];
+            }
+        }
+    });
 });
 
-const port = process.env.PORT || 3000;
```
Comment (arjunr00) [inline server/src/index.js:72]: put this back pls lmao

**Review by arjunr00 — CHANGES_REQUESTED** [review]

**Comment by DonLe27** [comment]
Superseding this branch with don/dogs pr

**Pull request closed without merging**